front/store/user.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+/**
+ * Default state object with initial values.
+ */
+const initialState = {
+  name: "Quentin",
```

Comment:
Dev Wavemind
dev@wavemind.ch

Reply:
Its just an example, it will be overwritten when we integrate the auth module that we made with Quentin